cli/lib/kontena/command.rb

```diff
@@ -188,6 +188,16 @@ def run(arguments)
     exit(@exit_code) if @exit_code.to_i > 0
     @result
   end
+
+  def subcommand_missing(name)
+    guesses = self.class.recognised_subcommands.flat_map(&:names).select {|sc_name| sc_name.start_with?(name)}
+    if guesses.size == 1
+      return self.class.find_subcommand(guesses.first).subcommand_class
+    elsif guesses.size > 1
+      puts "Suggestions: #{guesses.join(', ')}"
```

Comment:
This could just throw a nicer error directly: `ERROR: Non-unique subcommand prefix 'l' of list, ls, label`

instead of the current:

```
Suggestions: list, ls, label
ERROR: No such sub-command 'l'
```